spk/jellyfin/Makefile

```diff
@@ -14,7 +14,7 @@ MAINTAINER = stevenliuit
 DESCRIPTION = "The Free Software Media System. It is an alternative to the proprietary Emby and Plex."
 DISPLAY_NAME = Jellyfin
 STARTABLE = yes
-CHANGELOG = "1. Update jellyfin to 10.8.5 2.update dotnet 6.0.401"
+CHANGELOG = "1. Update jellyfin to 10.8.9"
```

Comment:
do you want to also consider updating the dotNET version to 6.0.405 similar to how the last update did? check out the file modifications at https://github.com/SynoCommunity/spksrc/pull/5438/files

Reply:
done